src/components/layout/Live.js

```diff
@@ -0,0 +1,125 @@
+import { React, useState } from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import { Button, Typography } from "@material-ui/core";
+
+import PeopleOutlineIcon from "@material-ui/icons/PeopleOutline";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  player: {
+    margin: theme.spacing(5),
+    marginBottom: theme.spacing(-5),
+    backgroundColor: "darkgrey",
+    height: "80vh",
+    width: "70vw",
+    borderRadius: "10px",
+    display: "flex",
+    flexDirection: "row-reverse",
+  },
+  status: {
+    flexGrow: 1,
+    // position: "fixed",
+    margin: "2vh",
+    display: "flex",
+    justifyContent: "center",
+    maxWidth: "80px",
+    maxHeight: "18px",
+    borderRadius: "12px",
+    padding: theme.spacing(0.3),
+    color: "white",
+    backgroundColor: "red",
+  },
+  contents: {
+    flexGrow: 1,
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    marginLeft: theme.spacing(5),
+    marginRight: theme.spacing(5),
+    marginTop: theme.spacing(7),
+    justifyContent: "space-between",
+  },
+  exit: {
+    textTransform: "none",
+    color: "white",
+    backgroundColor: "red",
+    minWidth: "80px",
+    borderRadius: "15px",
+    height: "30px",
+    display: "flex",
+    justifyContent: "center",
+    "&:hover": {
+      backgroundColor: "#d90000",
+    },
+  },
+  streamtitle: {
+    fontSize: "2.25rem",
+    fontWeight: 700,
+    color: "white",
+    paddingRight: "35vw",
+  },
+  viewcount: {
+    flexGrow: 1,
+    flexDirection: "row",
+    display: "flex",
```

Comment:
Material design or bootstrap layouts / grid classes would be easier to make components responsive than `display:  flex` with custom margins and width